XCUITests/SettingsTest.swift

```diff
@@ -7,34 +7,53 @@ import XCTest
 class SettingsTest: BaseTestCase {
     var navigator: Navigator!
     var app: XCUIApplication!
-
+    
     override func setUp() {
         super.setUp()
         app = XCUIApplication()
         navigator = createScreenGraph(app).navigator(self)
     }
-
+    
     override func tearDown() {
         super.tearDown()
     }
-
+    
     func testHelpOpensSUMOInTab() {
         navigator.goto(SettingsScreen)
         let appsettingstableviewcontrollerTableviewTable = app.tables["AppSettingsTableViewController.tableView"]
         appsettingstableviewcontrollerTableviewTable.cells["OpenWith.Setting"].swipeUp()
```

Comment:
Here, we should wait until the settings screen is fully drawn.

We have to wait until 'Use Compact Tabs' is visible.

waitforExistence(app.webViews.staticTexts["Use Compact Tabs"])